doc/man1/ciphers.pod

```diff
@@ -717,7 +717,7 @@ Set security level to 2 and display all ciphers consistent with level 2:
 
 =head1 SEE ALSO
 
-L<s_client(1)>, L<s_server(1)>, L<ssl(3)>
```

Comment:
Section 3 is wrong, it's in section 7 (have a look in `doc/man7`) 

I think I would prefer if the section number was changed unless there's a strong reason to change the reference.